http_server/http_server.go

```diff
@@ -0,0 +1,83 @@
+package httpserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/tomiok/fuego-cache/logs"
+	"net/http"
+	"strings"
+)
+
+type httpServer struct {
+	address string
+}
+
+type HttpResponse struct {
+	Data string `json:"data"`
+}
+
+type HttpRequestBody struct {
+	Value string `json:"value"`
+}
+
+func handleGet(w http.ResponseWriter, r *http.Request) {
```

Comment:
OK this. But (probably the issue is not well explained) we need one handler for getValue and another to setValue, then, mock the response since the operations in the cache are still in development. Although this seems mocked data, just improve the names and keep the basic handler to accept get and post in the same URI.

func GetValueHandler(w, r) {

}

func SetValueHandler(w,r) {

}

http.HandleFunc("/fuego/", CacheOperationHandler) //maybe another name